package.json

```diff
@@ -19,12 +20,16 @@
   "license": "MIT",
   "dependencies": {
     "debug": "^2.1.3",
+    "fs": "0.0.2",
```

Comment:
As a general note - this line here was missing on the earlier releases and caused a problem when I tried to use the latest `4.7.0` release in another project. It needs to be merged into **master** regardless of this PR.